examples/nocodb/README.md

```diff
@@ -0,0 +1,46 @@
+# What is this?
+
+This is an example [Opta](https://github.com/run-x/opta) configuration file to deploy [NocoDB](https://github.com/nocodb/nocodb) on AWS.
+
+You can also use this button to fill in the variable fields in the config.
+
+[![Deploy](https://raw.githubusercontent.com/run-x/opta/main/assets/deploy-to-aws-button.svg)](https://app.runx.dev/deploy-with-aws?url=https%3A%2F%2Fgithub.com%2Frun-x%2Fopta%2Fblob%2Fmain%2Fexamples%2Fnocodb%2Fnocodb-aws.yaml&name=NocoDB)
+
+# What does this do?
+It deploys a horizontally scalable NocoDB deployment on EKS in AWS. It uses AWS managed the RDS mysql instance. It also sets up various other resources like VPCs, subnets, load balancers etc.
+
+# Steps to deploy
+* Fill in the required variables in the config file, nocodb-aws.yaml.
+* Run `opta apply -c nocodb-aws.yaml` on the config file
+* You will be prompted to approve changes a couple of times as opta will go through various stages. This is expected, just keep approving.
+
+That's it. NocoDB is deployed on AWS. You can find the AWS load balancer URL to access the deployment by running `opta output` (looks for the field called `load_balancer_raw_dns`).
+
+**NOTE**: DNS and SSL/TLS is currently not set up. To get them to work, follow the next section.
+
+# Getting DNS to work
```

Comment:
Also needs a section on getting cloudfront to work

Reply:
Updated this. Can you check if anything needs to be changed.